JAMES-3711 Implement a Requeue mailet (#880)
apache/james-project: server/mailet/rate-limiter/README.adoc

</mailet>
----

When the rate is exceeded, all mails will be moved to processor *tooMuchMails* which send a bounce message notifies that user exceeded the rate limit.

=== Throttling

> Can combine RateLimit mailet with `Requeue` mailet, it allows for a (very basic) throttler: smooth the traffic to the expected rate.

Reviewer: ```suggestion
It's possible to combine RateLimit mailet with `Requeue` mailet to allow for a (very basic) throttler and smoothing the traffic to the expected rate.
```

The throttler work by re-enqueue mail.

> Requeue are supported configuration parameters:

Reviewer: ```suggestion
The supported configuration parameters for the `Requeue` mailet are:
```
